src/orquestra/sdk/schema/workflow_run.py

```diff
@@ -71,3 +71,14 @@ class WorkflowRun(WorkflowRunMinimal):
 class ProjectRef(BaseModel):
     workspace_id: WorkspaceId
     project_id: ProjectId
+
+
+class WorkspaceDef(BaseModel):
```

Comment:
As previously discussed in https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/109 `WorkspaceDef` and `ProjectDef` might not be the best names for these records. Usually we keep long-term support for types in `orquestra.sdk.schema` so after we make our mind it's gonna be set in stone. Let's catch up to figure out alternative naming together.